assets/js/common-components/b4/totals.tpl.liquid

```diff
@@ -0,0 +1,55 @@
+            <div class="col-12">
+                <hr class="mb-2 mt-0 border-1 border-gray-200" >
+            </div>
+            <dt ng-if-start="$ctrl.options.show.shipping" class="col-6" ng-init="title = '{{ 'totals.shipping' | t }}'" ng-include="'totals-title-template.tpl'"></dt>
+            <dd ng-if-end class="col-6">
+                <vc-totals-field-with-free field="$ctrl.order[$ctrl.fieldNames.shippingPrice] || $ctrl.order[$ctrl.fieldNames.shippingTotal]"></vc-totals-field-with-free>
+            </dd>
+            <div class="col-12">
+                <hr class="mb-2 mt-0 border-1 border-gray-200" >
+            </div>
+            <dt ng-if-start="$ctrl.options.show.payment" class="col-6 overflow-visible" ng-init="title = '{{ 'totals.paymentFees' | t }}'" ng-include="'totals-title-template.tpl'"></dt>
+            <dd ng-if-end class="col-6 fee-margin-bottom">
+                <vc-totals-field-with-free field="$ctrl.order[$ctrl.fieldNames.payment]"></vc-totals-field-with-free>
+            </dd>
+
+            {% unless settings.show_prices_with_taxes %}
+            <dt ng-if-start="$ctrl.options.show.taxes" class="col-6" ng-init="title = '{{ 'totals.taxes' | t }}'" ng-include="'totals-title-template.tpl'"></dt>
+            <dd ng-if-end class="col-6" ><span class="pull-right" ng-bind="$ctrl.order.taxTotal.formattedAmount"></span></dd>
+            {% endunless %}
+            <div class="col-12">
+                <hr class="mb-2 mt-0 border-1 border-gray-200" >
+            </div>
+        </dl>
+        <dl class="row mb-1">
+            <dt class="col"><span class="pull-left">{{ 'totals.total' | t }}</span></dt>
+            <dd class="col"><span class="pull-right font-weight-bold text-dark-green" ng-bind="$ctrl.order.total.formattedAmount"></span></dd>
+        </dl>
+        <div ng-if="$ctrl.options.show.outOfStockMessage && $ctrl.order.outOfStockError" class="bg-danger p-3 mb-3 rounded d-flex">
+            <i class="fa fa-times-circle fa-lg text-danger mr-2 mt-1"></i>
+            <small>{{ 'checkout.out_of_stock_error_message' | t }}</small>
+        </div>
+        <p class="mb-0" ng-if="!$ctrl.options.complete">
+            <small class="text-muted">{{ 'totals.estimated_fields' | t }} {{ 'totals.estimated_notice' | t }}</small>
+        </p>
+    </div>
+</div>
+
+<script id="totals-title-template.tpl" type="text/ng-template">
+    <span class="pull-left" ng-if="!$ctrl.options.complete"> {% raw %} {{ title }} {% endraw %} </span> {% if settings.show_prices_with_taxes %} {{ 'totals.withTax' | t }} {% endif %}
+</script>
+
+<script id="totals-field-with-free-template.tpl" type="text/ng-template">
+    <span ng-if="$ctrl.field.amount > 0" class="pull-right" ng-bind="$ctrl.field.formattedAmount"></span>
+    <span ng-if="$ctrl.field.amount <= 0" class="pull-right">{{ 'totals.free' | t }}</span>
+</script>
```

Comment:
May you format is with attributes-at-new-line style?
```html
<div class=""
     ng-init=""
```
Otherwise it's hard to read